astropy/stats/funcs.py

```diff
@@ -693,12 +695,13 @@ def poisson_conf_interval(n, interval='root-n', sigma=1, background=0,
            [  5.23161394,  11.94514152],
            [  6.05653896,  13.11020414]])
 
-    >>> poisson_conf_interval(7,
-    ...                       interval='frequentist-confidence').T
+    >>> poisson_conf_interval(
+    ...     7, interval='frequentist-confidence').T
```

Comment:
why change these?

Reply:
Pains my eyes to read. Purely stylistic.

Reply:
But, the whole line fits within 80 chars?

Reply:
This particular change is so that it would align with the rest of the multi-line examples, for consistency. Sorry, I meant you were referring to the entire docstring diff.

Reply:
I was indeed, then with a second look realized it's rather short anyway, so would fit. Somehow this whole black style putting everything flying in the next line does the same to me, as the other way around to you. But it's pure 🚲-shedding, so whichever you prefer is fine by me, as long as you don't put the closing parens on their own in a new line.